kernel/Makefile

```diff
@@ -149,9 +149,9 @@ show-tags: show-tag_$(2)$(3)$(4)
 # with DCT as part of the dependency on build_$(2)$(3)$(4) and then build
 # with DOCKER_CONTENT_TRUST explicitly set to 0
 
-# Skip perf build for now. See:
-# https://github.com/linuxkit/linuxkit/issues/3299
-ifeq ($(ARCH),SKIP)
+Only build perf on latest LTS and stable kernels.  Skip arm64.
```

Comment:
This line needs commenting out.

Reply:
Thanks for catching this.  I've fixedup and re-pushed as requested.